### PR TITLE
Adds Horn Option: Palmated Antlers

### PR DESCRIPTION
**Pull request by probbrie**
## About The Pull Request
Adds a new horn type, based on the wide palmated antlers of moose and elk!

## How This Contributes To The Skyrat Roleplay Experience
Helps cervid anthro players express their characters appearance better in-game. And allows intended character appearances to be better illustrated using in-game sprites.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>

image
</details>

## Changelog

:cl:
add: Adds new palmated antler horn type.
/:cl:

### CODE REVIEW

**Review by vinylspiders — APPROVED** [review]
Looks good to me, @GoldenAlpharex ?

File: modular_skyrat/master_files/icons/mob/sprite_accessory/horns_big.dmi
Comment (GoldenAlpharex) [inline modular_skyrat/master_files/icons/mob/sprite_accessory/horns_big.dmi:1]: If you don't have a use for the _BEHIND version, then it shouldn't be there as an empty icon state.

**Review by GoldenAlpharex — DISMISSED** [review]
Outside of that, looks great to me!

File: modular_skyrat/master_files/icons/mob/sprite_accessory/horns_big.dmi
Comment (probbrie) [reply modular_skyrat/master_files/icons/mob/sprite_accessory/horns_big.dmi:1]: Sorry, first time spriting with this system, it's fixed! Good catch.

**Pull request merged**